data/tsv/bdq_latn_phonemic.tsv

```diff
@@ -10,7 +10,6 @@ blok	b l ɔː k
 blu	b l uː
 băt	b a t
 bơbe	b ə b ɛː
-bơbư̆ng	b ə b ɨ ŋ
```

Comment:
Do we know these words should be excluded?

Reply:
I suppose the combining breve is allowed for whichever language this is, isn't it?

```
['LATIN SMALL LETTER B', 'LATIN SMALL LETTER O WITH HORN', 'LATIN SMALL LETTER B', 'LATIN SMALL LETTER U WITH HORN', 'COMBINING BREVE', 'LATIN SMALL LETTER N', 'LATIN SMALL LETTER G']
```